schema/component.schema.json

```diff
@@ -0,0 +1,142 @@
+{
+  "$anchor": "vimeo-component",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "type": "object",
+  "$merge": {
+    "source": {
+      "$ref": "component"
+    },
+    "with": {
+      "required": [
+        "_media"
+      ],
+      "properties": {
+        "_supportedLayout": {
+          "type": "string",
+          "title": "Supported layout",
+          "default": "both",
+          "_adapt": {
+            "editorOnly": true
+          }
+        },
+        "instruction": {
+          "type": "string",
+          "title": "Instruction",
+          "description": "This is the instruction text",
+          "default": "Select the play button to start the video.",
+          "_adapt": {
+            "translatable": true
+          }
+        },
+        "_setCompletionOn": {
+          "type": "string",
+          "title": "Set completion of Video on",
+          "description": "Defines what media event should trigger completion of this component.",
+          "default": "play",
+          "enum": [
+            "inview",
+            "play",
+            "ended"
+          ],
+          "_backboneForms": "Select"
+        },
+        "_media": {
+          "type": "object",
+          "title": "Media settings",
+          "properties": {
+            "_source": {
```

Comment:
Should the `_source` object have a placeholder like the [youtube](https://github.com/adaptlearning/adapt-youtube/blob/master/properties.schema#L60) version?

Reply:
Good call.